app/models/dfe_sign_in_user.rb

```diff
@@ -16,6 +32,7 @@ def self.begin_session!(session, omniauth_payload)
       'first_name' => omniauth_payload['info']['first_name'],
       'last_name' => omniauth_payload['info']['last_name'],
       'last_active_at' => Time.zone.now,
+      'id_token' => omniauth_payload['credentials']['id_token'],
```

Comment:
Can we come up with a more self-explanatory name? Would this be correct?

```suggestion
      'dfe_sign_in_sign_out_token' => omniauth_payload['credentials']['id_token'],
```